foundation-ui/utils/ReactUtil.js

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 
+function getUnwrappedElements(elements) {
+  if (Array.isArray(elements) && elements.length === 1
```

Comment:
Makes sense! I've done this offline, but I'll wait to push this until you and I can sync about the overall approach.